app/models/product_export_worker.rb

```
@@ -0,0 +1,14 @@
+class ProductExportWorker < ApplicationJob
+  def perform(products, product_export_id, user)
```

Suggestion: It may be a good idea to catch exceptions and report them to Sentry in this job so we become aware of any problems. Otherwise it will fail silently and the user will never receive the email they're expecting.

---

Done!

---

I think the exception capture should be in the job class to ensure it's only done in this context.